container/gitpod/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# hadolint ignore=DL3007
+FROM registry.opensuse.org/devel/openqa/ci/containers/base:latest
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# hadolint ignore=DL3004
+RUN sudo zypper -n in system-group-wheel npm && sudo zypper clean
+
+### Gitpod user ###
+# '-l': see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user
+
+# hadolint ignore=DL3004
+RUN sudo groupadd -g 33333 gitpod && \
+    sudo useradd -l -u 33333 -g 33333 -G wheel -md /home/gitpod -s /bin/bash -p gitpod gitpod && \
+    # passwordless sudo for users in the 'wheel' group
+    echo "%wheel ALL=(ALL) NOPASSWD: ALL" | sudo tee -a /etc/sudoers.d/00-wheel-group-nopasswd
+
+ENV HOME=/home/gitpod
+WORKDIR $HOME
+USER gitpod
+# custom Bash prompt
+RUN { echo && echo ". /etc/bash_completion.d/git-prompt.sh" && echo "PS1='\[\033[01;32m\]\u\[\033[00m\] \[\033[01;34m\]\w\[\033[00m\]\$(__git_ps1 \" (%s)\") $ '" ; } >> .bashrc
+
+### Gitpod user (2) ###
+# use sudo so that user does not get sudo usage info on (the first) login
+# hadolint ignore=DL3004
+RUN sudo echo "Running 'sudo' for Gitpod: success" && \
+    # create .bashrc.d folder and source it in the bashrc
+    mkdir -p /home/gitpod/.bashrc.d && \
+    (echo; echo "for i in \$(ls -A \$HOME/.bashrc.d/); do source \$HOME/.bashrc.d/\$i; done"; echo) >> /home/gitpod/.bashrc
+
+# explicitly set user/group IDs
+# hadolint ignore=DL3004
+RUN sudo mkdir -p /opt/testing_area && \
+    sudo chown -R "${NORMAL_USER}:users" /opt/testing_area
```

Comment:
IMHO this can be removed as well now as the base container instructions already do the same

Reply:
Indeed, that was oversight.